### ISSUE TITLE
Elections work stream: Compliance user interviews

### ISSUE THREAD

**Issue by PaulClark2** [18F/openFEC-web-app#2161]
This issue is a continuation of: https://github.com/18F/openFEC-web-app/issues/2100 

The focus of these interviews will be the compliance map users.

So that we can develop a better understanding of user needs regarding the Elections data and the previous Compliance map, conduct user interviews to collect the desired data, and then synthesize that data to identify themes that can (a) inform future designs, and/or (b) inform usability testing of existing tools.

Completion criteria:

 - [x] User interviews conducted
 - [x] Results synthesized and collected into a report

**Comment by nickykrause** [comment]
Wanted to drop an update here: I'm pretty sure that we are deferring the compliance user interviews to sometime after the 15th, which is after a filing deadline. Is that correct, @llienfec? If so, I am going to change the milestone for this task to Sprint 3.2.

**Comment by llienfec** [comment]
You're right @nickykrause.  These interviews are being moved back because the interviewees have a filing deadline on 7/15.

**Comment by patphongs** [comment]
Updating this: @llienfec will work on recruiting compliance folks for next sprint 3.3. I will also work on this too. cc @AmyKort

**Comment by AmyKort** [comment]
Is this ready to close?

**Comment by llienfec** [comment]
I think this is finished.  I don't think we plan, at this point, to probe the compliance map further.  We can always enter a new issue if we do.

**Comment by AmyKort** [comment]
Thanks. We'll revisit our findings when we start work again on the compliance map.

**Comment by jenniferthibault** [comment]
Connecting some GitHub dots for future reference. 

@JonellaCulmer  and I are trying to find the readout/summary of this research, and we enlisted @llienfec 's help:
- These interviews occurred on 8/3-4, in sprint 3.3 (8/1-8/14). 
- There's no mention of demoing the research in any of the following sprint demo documents
- We found a ["Homepage Data Compliance Themes" Google Doc](https://docs.google.com/document/d/1Ujhk9aj7o_bP4k03SW-AbicBh2u4IRRKb9l_l2a4uEs/edit?usp=sharing) 🔏  that provides some summary, but it is less comprehensive than we expected to find on this area, so we're unsure if that's the only summary documetn, or if there are more that we can't find right now. 

We'd like to check with @patphongs when she's back from leave after Dec 28 to see if there is further synthesis, and if so, connect it with this issue and organize it in our Google Drive structure so others can find it.
